library/src/main/java/com/github/pwittchen/reactivenetwork/library/ReactiveNetwork.java

```diff
@@ -101,11 +101,26 @@ public static ReactiveNetwork create() {
    * and false if not
    */
   public static Observable<Boolean> observeInternetConnectivity() {
-    return observeInternetConnectivity(DEFAULT_PING_INTERVAL_IN_MS, DEFAULT_PING_HOST,
+    return observeInternetConnectivity(DEFAULT_PING_INTERVAL_IN_MS, DEFAULT_PING_INTERVAL_IN_MS, DEFAULT_PING_HOST,
```

Comment:
You could create an additional private static final field called `DEFAULT_INITIAL_PING_INTERVAL_IN_MS` and use it here because it's related to a separate thing.